### ISSUE TITLE
Each frontend should be listening to a separate socket

### ISSUE THREAD

**Issue by achamayou** [microsoft/CCF#401]
As described in #237, our use of SNI makes using HTTP tooling difficult. We can make CCF easier to use by having a 1:1 mapping between listening sockets and frontends, and having a single identity for each one of them.

This depends on #400

**Comment by achamayou** [comment]
Now that all frontends use the same TLS identity, it is no longer clear that this is the way to go. Instead, we could replace SNI-based dispatch with a method prefix-based dispatch, ie method: "members/propose" goes to propose on the member frontend. This requires some refactoring of the frontend, but will make things nicer for REST integration.

Closing in favour of #406
